### DISCUSSION TITLE
Add a new payee through UI

### DISCUSSION THREAD

**Discussion by Go-rom** [mtotschnig/MyExpenses#1290] [Q&A] [answered]
Hello,

still testing Web UI (through my mobile hotspot wifi), i'm trying to add a new payee but the dedicated field seems to be only set as a list of the actual registered payees.
Am i missing something please ?

Thank you

ps : i've looked into the FAQ and issues about that but found nothing

**Comment by mtotschnig** [comment, marked as answer]
@Go-rom Thank you for bringing up this question. Up until recently the Web UI had a text input field with suggestions that allowed to create new payees. This was possible because previously payee names were unique so it was easy to map the input either to an existing payee or a create a new one. But since version 3.6.3, which introduced FinTS (allowing to connect with banks in Germany), there can be multiple payee with the same name, we now require the pair name + IBAN to be unique. So for the case of simplification, I decided to just send the list of existing payees to the browser, and have the user select one of them, requiring to use the app for creating new payees. I have created a ticket for bringing back the option to create new payees through web UI: https://github.com/mtotschnig/MyExpenses/issues/1312

**Reply by Go-rom** [reply]
Ok i fully understand the underlying technical choice.
For now i'll manage the payees through the app.
Thank your for the ticket, will monitor that.
